app/scripts/lib/tx-state-manager.js

```diff
@@ -0,0 +1,230 @@
+const extend = require('xtend')
+const ObservableStore = require('obs-store')
+const txStateHistoryHelper = require('./tx-state-history-helper')
+
+module.exports = class TransactionStateManger extends ObservableStore {
```

Comment:
i dont think if we should extend obs-store or just have one
the pattern so far is to have the controller and its memStore

Reply:
✔️